api/src/validators/words.js

```diff
@@ -0,0 +1,38 @@
+const v8n = require('v8n')
+
+const validateDict = v8n().passesAnyOf(
+  v8n().undefined(),
+  v8n().exact('wordnet'),
+  v8n().equal(''),
+)
```

Comment:
Here we prolly want to make sure `dict` is either  `'en-vi'` or `'wordnet'`. Tho technically we can always fallback to `en-vi` as the default dict. So somehow this validation is also redundant.